integration_tests/base_routes.py

```diff
@@ -714,6 +723,24 @@ async def async_auth(request: Request):
 # ===== Main =====
 
 
+def sync_without_decorator():
+    return "Success!"
+
+
+async def async_without_decorator():
+    return "Success!"
+
+
+app.add_route(HttpMethod.GET, "/sync/get/no_dec", sync_without_decorator)
+app.add_route(HttpMethod.PUT, "/sync/put/no_dec", sync_without_decorator)
+app.add_route(HttpMethod.POST, "/sync/post/no_dec", sync_without_decorator)
+app.add_route(HttpMethod.GET, "/async/get/no_dec", async_without_decorator)
+app.add_route(HttpMethod.PUT, "/async/put/no_dec", async_without_decorator)
+app.add_route(HttpMethod.POST, "/async/post/no_dec", async_without_decorator)
```

Comment:
Hey @IdoKendo 👋 

Thank you for the PR 😄 And apologies for the late review

I was wondering if we are exposing a public API for this, we should also allow strings as a method specifier.

e.g. `app.add_router("GET". "/sync/get")`

Reply:
Yeah, sounds good, added a consideration for it!